src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayTableViewCell.swift

```diff
@@ -28,26 +84,76 @@ class DiaryOverviewDayTableViewCell: UITableViewCell {
+		drawBorders(to: [.left, .right], on: testsStackView)
+
 		cellViewModel.diaryDayTests.forEach { diaryDayTest in
-			let testView = arrangedView(for: diaryDayTest)
-			testsStackView.addArrangedSubview(testView)
+
+			let containerView = UIView()
+			containerView.translatesAutoresizingMaskIntoConstraints = false
+
+			let imageView = UIImageView()
+			imageView.translatesAutoresizingMaskIntoConstraints = false
+			imageView.contentMode = .center
+			imageView.image = diaryDayTest.result == .negative ? UIImage(imageLiteralResourceName: "Test_green") : UIImage(imageLiteralResourceName: "Test_red")
+
+			let titleLabel = ENALabel()
+			titleLabel.style = .body
+			titleLabel.text = diaryDayTest.type == .pcr ? AppStrings.ContactDiary.Overview.Tests.pcrRegistered : AppStrings.ContactDiary.Overview.Tests.antigenDone
+
+			let detailLabel = ENALabel()
+			detailLabel.style = .subheadline
+			detailLabel.textColor = .enaColor(for: .textPrimary2)
+			detailLabel.text = diaryDayTest.result == .negative ? AppStrings.ContactDiary.Overview.Tests.negativeResult : AppStrings.ContactDiary.Overview.Tests.positiveResult
+
+			let verticalStackView = UIStackView(arrangedSubviews: [titleLabel, detailLabel])
+			verticalStackView.axis = .vertical
+			verticalStackView.spacing = 8.0
+
+			let horizontalStackView = UIStackView(arrangedSubviews: [imageView, verticalStackView])
+			horizontalStackView.translatesAutoresizingMaskIntoConstraints = false
+			horizontalStackView.alignment = .center
+			horizontalStackView.spacing = 15.0
+			containerView.addSubview(horizontalStackView)
+
+			NSLayoutConstraint.activate(
+				[
+					imageView.widthAnchor.constraint(equalToConstant: 32),
+					horizontalStackView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 18.0),
+					horizontalStackView.topAnchor.constraint(equalTo: containerView.topAnchor, constant: 12.0),
+					horizontalStackView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -8.0),
+					horizontalStackView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor, constant: -8.0)
+				]
+			)
+
+			drawBorders(to: [.top], on: containerView)
+			testsStackView.addArrangedSubview(containerView)
 		}
+		testsStackView.isHidden = cellViewModel.diaryDayTests.isEmpty
+	}
 
+	private func configureCheckinWithRisks(_ cellViewModel: DiaryOverviewDayCellModel) {
 		// Check-Ins with risk
-		checkinHistoryStackView.isHidden = cellViewModel.hideCheckinRisk
+		checkinsWithRiskStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+		checkinHistoryContainerStackView.isHidden = cellViewModel.hideCheckinRisk
+		checkinHistoryContainerStackView.backgroundColor = .enaColor(for: .darkBackground)
+		// Because we set the background color, the border of the underying view disappears. For this we need some new borders at the left and right.
```

Comment:
```suggestion
		// Because we set the background color, the border of the underlying view disappears. For this we need some new borders at the left and right.
```